src/components/ReserveForm.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { createNewReservation } from "../store/reservationReducer";
+import { fetchAllCars } from "../store/mainpage_reducer";
+import "./ReserveForm.css";
+
+const ReserveForm = () => {
```

Comment:
- [ ] If there is only one car in the form, I can't reserve any car.  

## Explanation

This is because the first car on the list gets autoselected in the form, but it acts as if the user didn't select anything (even if it's already selected in the form), so the JSON request does not include car_id:

![image](https://user-images.githubusercontent.com/57421823/200625746-92e524cc-9ffa-4f17-8d11-1cb76c685495.png)

## Tips and extra help

This is where the error lies:

![image](https://user-images.githubusercontent.com/57421823/200625898-92918590-ef78-4bc5-b8cf-97440b52b59d.png)

Please, kindly correct this bug so your app works as intended 💪